### PR TITLE
Added yml file corresponding to running unit tests in both windows, ubuntu and osx.

### PR DESCRIPTION
**Pull request by MokoSan**
Added yml file corresponding to running unit tests via github workflows in both windows, ubuntu and osx. These test will run for every push and on pull request creation for all branches.

### CODE REVIEW

**Comment by Maoni0** [comment]
hi @MokoSan, I don't think this needs updates after [this PR](https://github.com/Maoni0/realmon/pull/5) was merged, correct (since the test dir didn't change)? if you could please confirm I will merge this one :)

**Comment by MokoSan** [comment]
Hey @Maoni0, this PR automates running all the unit tests right after a dev creates a pull request or pushes new changes to all branches to make sure the unit test runs are automated. Please let me know if this isn't needed; I think from the perspective of automated unit testing, it might help capture errors earlier than later. 

An example of a run is:
https://github.com/MokoSan/realmon/runs/4188326247?check_suite_focus=true 

Thanks,
M.

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+on:
+  push:
+  pull_request:
+    branches:
+      - '*' 
+    paths:
+      - '**UnitTests.csproj'
+
+env:
+  DOTNET_VERSION: '5.0.301' # The .NET SDK version to use
```
Comment (hez2010) [inline .github/workflows/run_tests.yml]: We have bumped dotnet version to `6.0.100` in the latest commit.

```suggestion
  DOTNET_VERSION: '6.0.100' # The .NET SDK version to use
```

**Review by hez2010 — CHANGES_REQUESTED** [review]

**Comment by Maoni0** [comment]
yep, I got that the purpose was adding the tests to run automatically and thought it was great to add them. just wanted to make sure that it didn't need any updates after the last commit. I saw that @hez2010 already helped out with that. so that's great! I'll merge. thanks very much for this!

**Comment by MokoSan** [comment]
> yep, I got that the purpose was adding the tests to run automatically and thought it was great to add them. just wanted to make sure that it didn't need any updates after the last commit. I saw that @hez2010 already helped out with that. so that's great! I'll merge. thanks very much for this!

Thank you @Maoni0 and @hez2010!

**Pull request merged**